core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -84,7 +84,9 @@ object ApiVersion {
     KAFKA_2_1_IV2,
     // Introduced broker generation (KIP-380), and
     // LeaderAdnIsrRequest V2, UpdateMetadataRequest V5, StopReplicaRequest V1
-    KAFKA_2_2_IV0
+    KAFKA_2_2_IV0,
+    // New error code for ListOffsets when a new leader is lagging behind former HW (KIP-207)
+    KAFKA_2_2_IV1
```

Comment:
I don't think a new ApiVersion is needed here because we're not actually changing anything for inter-broker communication.  Also KAFKA_2_2_IV0 hasn't yet been released.

Reply:
This is the conventional approach. We use the internal versions in case people are running from trunk.

Reply:
As I understand this versioning scheme, the `_IV#` versions never get released, right? They are like incrementing snapshot versions of the protocol.